libraries/Microsoft.Bot.Schema/Teams/Generated/TabResponseCard.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Schema.Teams
+{
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+    
+    /// <summary>
+    /// Envelope for cards for a Tab request.
+    /// </summary>
+    public partial class TabResponseCard
```

Comment:
>partial  [](start = 11, length = 8)

Why is this partial?

Reply:
Many/most of the .schmea files are partial classes, and have a CustomInit which can be called during construction.  I think this is part of the ServiceClient pattern.